style.css

```diff
@@ -0,0 +1,635 @@
+    padding: 0;
+    margin: 0;
+  }
+
+  .first {
+    font-weight: 400;
+    padding-top: 10px;
+    color: #ec5242;
+    margin: 0;
+  }
+
+  .second {
+    font-weight: 900;
+    color: #ec5242;
+    margin: 20px;
+    font-size: 25px;
+    margin-bottom: 15px;
+  }
+
+  .second-pr {
+    margin: 20px;
+    font-weight: 400;
+    color: #272a31;
+    margin-bottom: 15px;
+  }
+
+  .dates {
+    font-weight: 700;
+    margin: 20px;
+  }
+
+  .menu {
+    cursor: pointer;
+    align-self: center;
+    display: none;
+  }
+
+  .bar {
+    width: 15px;
+    margin: 4px auto;
+    background-color: black;
+    height: 2px;
+    display: none;
+  }
+}
+
+.card-section {
+  background-color: #272a31;
+  margin: 0;
+  padding: 20px 0;
+}
+
+.card-main {
+  align-content: center;
+  color: #fff;
+  text-align: center;
+  margin: 0;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 6px;
+  justify-content: center;
+  margin-bottom: 10px;
+}
+
+.cd-card {
+  display: flex;
+  gap: 20px;
+  background: #6c757d;
+  width: 100%;
+  align-items: center;
+}
+
+.fa-solid {
+  font-size: 30px;
+  width: 10%;
+}
+
+.fa-brands {
+  font-size: 30px;
+  width: 10%;
+}
+
+.cards-hd {
+  color: #ec5242;
+  font-size: 15px;
+  width: 30%;
+}
+
+.cards-pr {
+  color: #fff;
+  font-size: 2px;
+  font-weight: 100;
+  width: 55%;
+}
+
+.main-btn-2022 {
+  text-align: center;
+}
+
+.card-btn {
+  color: #fff;
+  text-align: center;
+  display: none;
+}
+
+.btn-2022 {
+  background: #ec5242;
+  padding: 10px;
+}
+
+@media (min-width: 768px) {
+  .card-section {
+    margin: 0;
+  }
+
+  .cards {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .cd-card {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    width: 18%;
+    padding-top: 10px;
+  }
+
+  .btn-2022 {
+    display: none;
+  }
+}
+
+.speakers-section {
+  background-color: #d3d3d3;
+}
+
+.spks-main {
+  display: flex;
+  flex-direction: column;
+}
+
+.speakers {
+  display: flex;
+  margin: 0;
+  align-items: center;
+  padding: 0;
+  flex-direction: row;
+  gap: 14px;
+  width: 100%;
+}
+
+.speaker-pr {
+  margin: 0;
+  padding: 10px;
+}
+
+.h2-h2 {
+  font-size: 20px;
+}
+
+.h3-h3 {
+  font-size: 16px;
+  font-weight: 500;
+  color: #ec5242;
+}
+
+.p2-p3 {
+  font-size: 8px;
+  font-weight: 100;
+  color: #272a31;
+}
+
+.body-p2 {
+  width: 70%;
+}
+
+.img-tg {
+  width: 20%;
+}
+
+.img-tg img {
+  width: 100%;
+}
+
+.partners-section {
+  background-color: #272a31;
+  padding: 0;
+  margin: 0;
+}
+
+.h-ptns {
+  align-content: center;
+  color: #d3d3d3;
+  text-align: center;
+  margin: 0;
+}
+
+.ul-list {
+  display: flex;
+  list-style: none;
+  gap: 50px;
+  color: #d3d3d3;
+  justify-content: center;
+  text-align: center;
+  margin: 0;
+  padding: 10px;
+}
+
+.li-li {
+  padding: 5px;
+}
+
+.rights-section {
+  display: flex;
+  gap: 30px;
+  align-content: center;
+  background-color: #d3d3d3;
+  height: auto;
+  justify-content: center;
+}
```

Comment:
- [ ] Dear @monicah79, As the previous code reviewer mention, the background color for the footer should be white on the desktop version and black on the mobile version.

## Figma design or desktop and mobile 
![footer](https://user-images.githubusercontent.com/69077061/204107445-2bf8f0fb-c074-4abc-a5c4-983f237baf98.PNG)
![footer-m](https://user-images.githubusercontent.com/69077061/204107446-b42fe23b-2dad-4210-96bf-42c21361b95e.PNG)